src/watts/plugin_sas.py

```diff
@@ -0,0 +1,203 @@
+# SPDX-FileCopyrightText: 2022 UChicago Argonne, LLC
+# SPDX-License-Identifier: MIT
+
+from contextlib import redirect_stdout, redirect_stderr
+from datetime import datetime
+from pathlib import Path
+import shutil
+import time
+from typing import List, Optional
+
+import os
+import glob
+import numpy as np
+import pandas as pd
+import platform
+
+from .fileutils import PathLike, run as run_proc, tee_stdout, tee_stderr
+from .parameters import Parameters
+from .plugin import TemplatePlugin
+from .results import Results
+
+
+class ResultsSAS(Results):
+    """SAS simulation results
+
+    Parameters
+    ----------
+    params
+        Parameters used to generate inputs
+    time
+        Time at which workflow was run
+    inputs
+        List of input files
+    outputs
+        List of output files
+
+    Attributes
+    ----------
+    stdout
+        Standard output from SAS run
+    csv_data
+        Dictionary with data from .csv files
+    """
+    def __init__(self, params: Parameters, time: datetime,
+                 inputs: List[PathLike], outputs: List[PathLike]):
+        super().__init__('SAS', params, time, inputs, outputs)
+        self.csv_data = self._save_sas_csv()
+
+    @property
+    def stdout(self) -> str:
+        return (self.base_path / "SAS_log.txt").read_text()
+
+    def _save_sas_csv(self) -> dict:
```

Comment:
To me, the word "save" means we should be "saving" something (to disk). I would suggest renaming this to `_get_sas_csv_data` or something along those lines